server/lib/dataone/fakeoauth.py

```diff
@@ -0,0 +1,46 @@
+from urllib.parse import urlparse, urlunparse, quote
+from d1_common.env import D1_ENV_DICT
+
+from girder.api.rest import getApiUrl
+from girder.plugins.oauth.providers.base import ProviderBase
+
+# Because, why would they keep up to date list of CNs and deployments?
+D1_ENV_DICT["stage2"] = dict(base_url="https://cn-stage-2.test.dataone.org/cn")
```

Comment:
Original URL may be missing `/v2` at the end, as it is present here: https://github.com/whole-tale/dashboard/blob/master/app/components/dashboard/run/left-panel/component.js#L321